cmd/cri-containerd/cri_containerd.go

```diff
@@ -200,3 +216,23 @@ func setGLogLevel(l logrus.Level) error {
 	}
 	return nil
 }
+
+// loadCgroup loads the cgroup associated with path if it exists and moves the current process into the cgroup. If the cgroup
+// is not created it is created and returned.
```

Comment:
If the cgroup status for the path is cgroupdeleted an attempt to create the cgroup is made and if successful the current process is moved into the cgroup.

Reply:
Will do.